### DISCUSSION TITLE
请问一下,这个验证码识别错误该如何解决

### DISCUSSION THREAD

**Discussion by L1uYun** [Mufanc/iSmartAuto#47] [General]
![B_`DMZ4V~D6 HD @J5S}LE](https://github.com/Mufanc/iSmartAuto/assets/119873107/482b93bf-ac11-4e00-9bef-1ca95d0e8a50)

**Comment by L1uYun** [comment]
改成手动输入验证码,然后问题解决了

**Reply by L1uYun** [reply]
等有空学学过验证码的机制,然后再改改这个代码

**Reply by MagicalLeo** [reply]
請問如何改成手動輸入呢
